src/main/java/Storage.java

```diff
@@ -0,0 +1,92 @@
+import objects.Deadline;
+import objects.Event;
+import objects.Task;
+import objects.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+
+    /**
+     * This function reads the text file at the specified file path and parses the tasks,
+     * adding them into list of tasks in the app. It allows for persistent storage of tasks.
+     *
+     * @param FILE_PATH path of the txt file that contains data of tasks
+     * @param tasks list of Task objects of the app
+     * @throws FileNotFoundException exception thrown when the txt file is not found
+     */
+    public void loadTasks(String FILE_PATH, List<Task> tasks) throws FileNotFoundException {
+        File dataFile = new File(FILE_PATH);
+        Scanner sc = new Scanner(dataFile);
+        while (sc.hasNext()) {
+            String[] dataChunk = sc.nextLine().split(" \\| ");
+            switch (dataChunk[0]) {
+            // Task is a Todo
+            case "T": {
+                // "1" means done, "0" means not done
+                Boolean isDone = dataChunk[1].equals("1");
+                String name = dataChunk[2];
+                tasks.add(new Todo(name, isDone));
+                break;
```

Comment:
Maybe you can consider putting this logic in an overloaded Todo constructor instead?